### PR TITLE
make  DBRecoveryMode comparable

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
make  DBRecoveryMode comparable

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
@siddontang @BusyJay PTAL

**Comment by BusyJay** [comment]
LGTM

File: librocksdb_sys/build.sh [hunk 1]
```diff
@@ -105,11 +105,11 @@ function compile_rocksdb() {
         return
     fi
 
-    version=v4.9
-	vernum=4.9
+    version=4.12.fb
+	vernum=4.12.fb
```
Comment (siddontang) [inline librocksdb_sys/build.sh]: alignment ?

**Comment by siddontang** [comment]
Rest LGTM

File: librocksdb_sys/build.sh (hunk 1 above)
Comment (zhangjinpeng1987) [inline librocksdb_sys/build.sh]: Maybe git mistake.

**Comment by zhangjinpeng1987** [comment]
@siddontang PTAL again

**Comment by siddontang** [comment]
LGTM

**Pull request merged**